jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/vo/PermissionVH.java

```diff
@@ -0,0 +1,22 @@
+package com.tmobile.pacbot.azure.inventory.vo;
+
+public class PermissionVH {
```

Comment:
move it to batch-commons instead of creating same file twice.

Reply:
moved